README.md

```diff
@@ -75,13 +75,14 @@ let zfsDatasetsCommand:Command = Command(bash:"zfs list -t dataset")
 let zfsProcessInterface = ProcessInterface(command:zfsDatasetsCommand, stdout:.active(.lf), stderr:.active(.unparsedRaw))
 
 //launch the process. if you are running many concurrent processes (using most of the available resources), this is where your process will be queued until there are enough resources to support the launched process.
-let outputStreams = try await zfsProcessInterface.launch()
+try await zfsProcessInterface.launch()
 
 //handle lines of stdout as they come in
 var datasetLines = [Data]()
 for await outputLine in await zfsProcessInterface.stdout {
-	print("dataset found: \( String(data:outputLine, encoding:.utf8) )")
-	datasetLines += outputLine
+	guard let outputLineString = String(data: outputLine, encoding: .utf8) else { continue }
+	print("dataset found: \(outputLineString)")
```

Comment:
Added a guard statement here to get around Xcode warning about the string being an optional.

<img width="973" alt="CleanShot 2023-07-27 at 14 14 21@2x" src="https://github.com/tannerdsilva/SwiftSlash/assets/923876/41e31bd0-f143-4f6e-a351-6947f3182b20">

> String interpolation produces a debug description for an optional value; did you mean to make this explicit?